### PR TITLE
[vested-deversifi] Strategy for allowing those with vested deversifi tokens on-chain to vote

### PR DESCRIPTION
**Pull request by plutoegg**
Changes proposed in this pull request:
- New strategy for voting with vested tokens where the vesting contract implements a 'delegateForVoting' function returning an address

### CODE REVIEW

File: src/strategies/vested-deversifi/index.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+const vestedABI = [
+  {
+    inputs: [],
+    name: 'delegateForVoting',
+    outputs: [{ internalType: 'address', name: '', type: 'address' }],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/vested-deversifi/index.ts:15]:
```suggestion
const vestedABI = [ 'function delegateForVoting() view returns (address)' ];
```

File: src/strategies/vested-deversifi/index.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+const xDVFABI = [
+  {
+    inputs: [{ internalType: 'address', name: 'account', type: 'address' }],
+    name: 'balanceOf',
+    outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/vested-deversifi/index.ts:25]:
```suggestion
const xDVFABI = [ 'function balanceOf(address account) view returns (uint256)' ];
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by plutoegg** [comment]
Hi @ChaituVR formatting of the ABIs has been resolved :)

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Strategy is live here: https://snapshot.org/#/strategy/vested-deversifi

**Pull request merged**